fenix/app/metrics.yaml

```diff
@@ -10828,12 +10777,170 @@ shopping.settings:
     data_sensitivity:
       - interaction
     notification_emails:
       - android-probes@mozilla.com
-      - cgordon@mozilla.com
-    expires: 131
+    expires: never
     metadata:
       tags:
         - Shopping
+fx_suggest:
+  ping_type:
+    type: string
+    description: >
+      The ping's type. In other situations might be designated by an event's
+      name or an interaction field. E.g. "fxsuggest-impression",
+      "fxsuggest-block", "fxsuggest-click".
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1857092
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3958#issuecomment-1758607927
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - lina@mozilla.com
+      - ttran@mozilla.com
+      - najiang@mozilla.com
+    expires: never
+    send_in_pings:
+      - fx-suggest
+  source:
+    type: string
+    description: |
+      The source of the interaction. E.g. "urlbar".
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1857092
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3958#issuecomment-1758607927
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - lina@mozilla.com
+      - ttran@mozilla.com
+      - najiang@mozilla.com
+    expires: never
+    send_in_pings:
+      - fx-suggest
+  match_type:
+    type: string
+    description: >
+      Whether this was a best/top match or not. Either "best-match" or
+      "firefox-suggest".
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1857092
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3958#issuecomment-1758607927
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - lina@mozilla.com
+      - ttran@mozilla.com
+      - najiang@mozilla.com
+    expires: never
+    send_in_pings:
+      - fx-suggest
+  block_id:
+    type: string
+    description: |
+      A unique identifier for the suggestion (a.k.a. a keywords block).
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1857092
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3958#issuecomment-1758607927
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - lina@mozilla.com
+      - ttran@mozilla.com
+      - najiang@mozilla.com
+    expires: never
+    send_in_pings:
+      - fx-suggest
+  advertiser:
+    type: string
+    description: |
+      The name of the advertiser providing the sponsored Suggestion
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1857092
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-android/pull/3958#issuecomment-1758607927
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - lina@mozilla.com
+      - ttran@mozilla.com
+      - najiang@mozilla.com
+    expires: never
+    send_in_pings:
+      - fx-suggest
+  is_clicked:
```

Comment:
Let's omit `is_clicked` for now, and add it when we add impressions.

[IIUC from this code](https://searchfox.org/mozilla-central/rev/81f368dab93fff035ce7fcc376e16990e89dd5ec/browser/components/urlbar/UrlbarProviderQuickSuggest.sys.mjs#744-757), when a sponsored suggestion is clicked, we send both an impression (`fxsuggest-impression`) ping with `is_clicked: true` _and_ an `fxsuggest-click` ping—so `is_clicked` (ironically! 😅) isn't meaningful for click pings, and we don't have impressions yet.

Reply:
okie doke